tests/e2e/specs/admin/1-activate-extension.test.js

```diff
@@ -8,8 +8,12 @@ describe( 'Store admin can login and make sure WooCommerce Shipping & Tax extens
 		const slug = 'woocommerce-services'
 		await StoreOwnerFlow.login();
 		await StoreOwnerFlow.updateWPDB();
+		await StoreOwnerFlow.openSettings();
```

Comment:
```suggestion
		await StoreOwnerFlow.openSettings(); // Navigate to a WP admin page to allow DB migrations to run prior to doing assertions below.
```

Reply:
I have added the comment. But with a different wording. LMKWYT?